sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala

```diff
@@ -362,7 +362,7 @@ object DottyPlugin extends AutoPlugin {
 
       // Configuration for the doctool
       resolvers ++= (if(!useScala3doc.value) Nil else Seq(Resolver.jcenterRepo)),
-      useScala3doc := false,
+      useScala3doc := true,
```

Comment:
To keep new versions of sbt-dotty working with old compilers, I would prefer if this was conditional on the compiler version:
```suggestion
      useScala3doc := VersionNumber(scalaVersion.value).matchesSemVer(SemanticSelector(">=3.0.0-M3")),
```
(Semantic versioning API from https://www.scala-sbt.org/1.x/docs/sbt-1.2-Release-Notes.html#Semantic+Version+selector+API)